src/simplewallet/simplewallet.cpp

```diff
@@ -101,6 +101,7 @@ enum TransferType {
 
 namespace
 {
+  const auto allowed_priority_strings = {"default", "unimportant", "normal", "elevated", "priority"};
```

Comment:
@Detegr After v5 hardfork GUI will use Low (x1), Default (x4), Medium (x20) and High (x166)
see https://github.com/monero-project/monero-core/pull/597